code/style.css

```diff
@@ -2,6 +2,195 @@
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
+*{
```

Comment:
In generell I like very much how you styled the page; it gives a nice picture and the colors work perfectly